tools/objc/base.py

```diff
@@ -74,6 +74,7 @@ def convert_types(config, package, type):
 
     @staticmethod
     def convert_type(config, package, type):
+        type_info = type
```

Comment:
It's to work around the potential reuse of type when 'type = type.name.value' is executed. I still need access type.type_arguments later on.

Rasekh had the same question, and I didn't intend this to necessarily be the optimal refactor. But there a number of ways to go, so this at least raises the question. Let me know what you want to do.